src/orderbook.js

```diff
@@ -1197,6 +1336,44 @@ OrderBook.prototype.computeAutobridgedOffers = function() {
   this._offersAutobridged = autobridgeCalculator.calculate();
 };
 
+OrderBook.prototype.computeAutobridgedOffersWrapper = function() {
+  var startTime = Date.now();
+  this.computeAutobridgedOffers();
+  this.mergeDirectAndAutobridgedBooks();
+  var lasted = (Date.now() - startTime);
+  
+  const newMult =
+    ((lasted * 2 / OrderBook.AUTOBRIDGE_CALCULATE_THROTTLE_TIME) << 0) + 1;
```

Comment:
> I don't think we can rely on this this results - it shows me 25% slower on one run, and 6% faster on second.

Maybe you have other processes running on your system? It is consistent for me.

> Thant's the bad thing in js - I want integer arithmetic here, I don't want floats.

I don't follow this. The only difference is whether it gets truncated to 32 bits right?